### PR TITLE
Author 006 a signuplogin ui

### PR DESCRIPTION
**Pull request by forgeno**
Created registration and login UI. Hooked into login endpoints to authorize login properly.

### CODE REVIEW

File: frontend/src/pages/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    shouldRender = () => {
+        if (this.state.showLogin) {
+            return <LoginFormComponent changePage={this.switchLoginContents}/>
+        } else {
+            return <RegisterFormComponent changePage={this.switchLoginContents}/>
+        }
+    }
+
+    finishAnimation = () =>{
```
Comment (derrick-w) [inline frontend/src/pages/Login.jsx]: Remove animation finish check

**Review by derrick-w — CHANGES_REQUESTED** [review]

**Review by htruong4 — DISMISSED** [review]
can you removed the comment?

**Review by derrick-w — APPROVED** [review]

**Review by shchau — COMMENTED** [review]
image

**Review by htruong4 — APPROVED** [review]
Looks good.

**Pull request merged**